style.css

```diff
@@ -119,66 +126,63 @@ h3,
+  transition: background-color 0.5s ease;
+}
 
 .home:not(.scrolled) .top-navigation {
-    -webkit-box-shadow: none;
-            box-shadow: none; /* drop default shadow because it'll look odd with transparent nav menu bar */
-    --color-background-navigation_bar: transparent;
-  }
-
-/* override P4 default: var(--white); */
+  -webkit-box-shadow: none;
+          box-shadow: none; /* drop default shadow because it'll look odd with transparent nav menu bar */
+  --color-background-navigation_bar: transparent; /* override P4 default: var(--white); */
+}
 
 .home:not(.scrolled) .top-navigation .nav-menu {
-      --color-text-nav_link: var(--wp--preset--color--white); /* override P4 default: var(--grey-900); */
-    }
+  --color-text-nav_link: var(--wp--preset--color--white); /* override P4 default: var(--grey-900); */
+}
 
 .home:not(.scrolled) .top-navigation .nav-submenu {
-      --submenu-nav-link--color: var(--grey-900);
-      --submenu-nav-link--hover--color: var(--submenu-nav-link--color);
-    }
+  --submenu-nav-link--color: var(--grey-900);
+  --submenu-nav-link--hover--color: var(--submenu-nav-link--color);
+}
 
 .home:not(.scrolled) .top-navigation #search_form #search_input::-webkit-input-placeholder {
-        color: var(--wp--preset--color--white);
-      }
+  color: var(--wp--preset--color--white);
+}
 
 .home:not(.scrolled) .top-navigation #search_form #search_input::-moz-placeholder {
-        color: var(--wp--preset--color--white);
-      }
+  color: var(--wp--preset--color--white);
+}
 
 .home:not(.scrolled) .top-navigation #search_form #search_input:-ms-input-placeholder {
-        color: var(--wp--preset--color--white);
-      }
+  color: var(--wp--preset--color--white);
+}
 
 .home:not(.scrolled) .top-navigation #search_form #search_input::-ms-input-placeholder {
-        color: var(--wp--preset--color--white);
-      }
+  color: var(--wp--preset--color--white);
+}
 
 .home:not(.scrolled) .top-navigation #search_form #search_input, .home:not(.scrolled) .top-navigation #search_form #search_input::placeholder {
-        color: var(--wp--preset--color--white);
-      }
+  color: var(--wp--preset--color--white);
+}
 
 .home:not(.scrolled) .top-navigation #search_form button .icon {
-          fill: var(--wp--preset--color--white);
-        }
+  fill: var(--wp--preset--color--white);
+}
 
 .home .site-logo img {
-      -webkit-transition: height 0.4s;
-      transition: height 0.4s;
-      /* bump height to get big logo (bumped up by fraction of golden ratio) */
-      height: -webkit-calc(26px + 1/2 * -webkit-calc(1.61803 * 26px - 26px));
-      height: calc(26px + 1/2 * calc(1.61803 * 26px - 26px));
-    }
+  -webkit-transition: height 0.4s;
+  transition: height 0.4s;
+  /* bump height to get big logo (bumped up by fraction of golden ratio) */
+  height: 34.03439px;
+}
 
-body.scrolled.home .site-logo img {
-        height: 26px; /* default P4 value */
-      }
+.home .site-logo img body.scrolled.home .site-logo img {
```

Comment:
This doesn't seem right does it?

![image](https://github.com/greenpeace/planet4-child-theme-australiapacific/assets/1072292/e447a594-fb22-4db6-90a5-f34ac4920720)

Reply:
Okay, I've fixed this now!